### DISCUSSION TITLE
Which file to modify to simulate a shut-down ?

### DISCUSSION THREAD

**Discussion by ACH52100** [OpenFAST/openfast#1415] [Q&A]
Hello !

Currently I am trying to simulate a normal shut down on 5MW offshore wind turbine. I ve followed the intructions given on this [page](https://openfast.readthedocs.io/en/main/_downloads/d8bd014121d6505cb25cf49bee5eaa80/Old_FAST6_UsersGuide.pdf) . The problem is that I don't find the file containing the parameters to change. As example, within the ServoDyn file, I see the parameters "PCMode" or "TPitManS" to change, but not the "TPitManE" one. So, I don't know if I am looking the wrong file or if my ServoDyn file is incomplete. I put 2 screen shots to illustrate the situation.
![image](https://user-images.githubusercontent.com/123339005/213950331-89ad410e-d1d7-4afe-8eb3-e03186cc00b2.png)
![image](https://user-images.githubusercontent.com/123339005/213950344-e0e8054a-3046-441d-ba93-d2dd5bc814e0.png)

Thank you for your help ! :)

**Comment by jjonkman** [comment]
Dear @ACH52100,

What used to be `TPitManE` in FAST v6 was replaced with `PitManRat` in FAST v8 and OpenFAST.  Typically, when the safety system overrides the operational controller, it will command a given pitch rate.  We found it difficult to set `TPitManE` because the actual pitch angle when time `TPitManS` is reached is generally unknown, so, it was not possible to calculate `TPitManE` based on this known pitch rate.  So, we decided to replace the `TPitManE` with the rate of the override pitch maneuver (`PitManRat`), which should be easier to set.  This is mentioned in the FAST v8 ReadMe file: https://openfast.readthedocs.io/en/main/_downloads/5f2ddf006568adc9b88d8118dc3f1732/FAST8_README.pdf.

Best regards,

**Reply by ACH52100** [reply]
Thank you very much for the explanations ! Now everything is clear in my mind ! :D
